user/super/com/google/gwt/emul/java/text/Normalizer.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2024 Google Inc.
```

Comment:
```suggestion
 * Copyright GWT Project Authors
```

see https://github.com/gwtproject/gwt/commit/3e8458260cae857560d8e94cf445c67bc3274954